src/dashboard-widgets/patient-info-header/patient-info-header.parcel.tsx

```diff
@@ -192,7 +214,7 @@ function PatientInfoHeaderParcel(props: PatientInfoHeaderParcelProps) {
   }
 
   function handleOnclickEditAddress(event) {
-    const redirect_pat_url = `https://openmrs-spa.org/openmrs/registrationapp/editSection.page?patientId=${props.patientUuid}&sectionId=contactInfo&appId=referenceapplication.registrationapp.registerPatient&returnUrl=/openmrs/coreapps/clinicianfacing/patient.page?patientId=${props.patientUuid}&`;
+    const redirect_pat_url = `https://openmrs-spa.org/openmrs/spa/patient-dashboard/${props.patientUuid}/contact-information`;
     event.preventDefault();
     window.location.href = redirect_pat_url;
```

Comment:
This probably causes a full page reload, right? Do this instead: `window.singleSpaNavigate(redirect_pat_url)` and it will avoid a page reload